### DISCUSSION TITLE
自动上传截图到默认图床

### DISCUSSION THREAD

**Discussion by SUPMColdRain** [Molunerfinn/PicGo#755] [Ideas]
> 请问能否像OneDrive一样，添加一个自动把截图上传到默认图床的功能？🤔

本人使用markdown编辑器，经常需要把截图粘贴到编辑器中。
如果能自动把截图上传到这样，原来截图后直接粘贴的行为，就会变成截图后直接粘贴图床中图片链接的行为。

不知道这项功能能否在下一次更新里实现？

**Comment by Molunerfinn** [comment]
electron未提供系统截图事件，要实现需要调用系统底层接口，成本比较大，未来会研究研究
